src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -42,7 +43,7 @@ def remapDBS3Keys(data, stringify=False, **others):
                'block_name': 'BlockName', 'lumi_section_num': 'LumiSectionNumber'}
 
     mapping.update(others)
-    formatFunc = lambda x: encodeUnicodeToBytes(x) if stringify else x
+    formatFunc = lambda x: encodeUnicodeToBytes(x) if PY2 and stringify else x
```

Comment:
Shall we replace it by the awesome(!) conditional encodeUnicode function you created yesterday? Don't forget importing that though.

Reply:
Sure! I think this was a relic from before merging `encodeUnicodeToBytesConditional` in master, thanks for pointing this out!

Reply:
Which one do you prefer?

```python
formatFunc = lambda x: encodeUnicodeToBytesConditional(x, condition=PY2 and stringify)
formatFunc = partial(encodeUnicodeToBytesConditional, condition=PY2 and stringify)
```

Reply:
Is this `partial` a method from the `functools` module? I guess I had not seen it before. If so, since it's a new import, I'd go with the old and good lambda function.

Reply:
yep, its `functools.partial`, but I agree that with the lambda it's more clear. Thanks!